bin/internal.js

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('../lib/config').root();
+const db = require('../lib/db');
+const logger = require('../lib/logging')('bin.internal');
+const server = require('../lib/server/clients').create();
+
+logger.debug('config', config);
+db.ping().done(function() {
+  server.start(function() {
+    logger.info('listening', server.info.uri);
```

Comment:
Oh, actually I guess this will come out for free because we pass the name to the logger, never mind :-)